### PR TITLE
Update cost_details.md

### PR DESCRIPTION
**Pull request by plaxidesnjeridd**
Multi-month doc update

### Merge instructions

- [ ] Please merge after reviewing

### CODE REVIEW

File: content/en/account_management/plan_and_usage/cost_details.md [hunk 1]
```diff
@@ -46,17 +45,18 @@ The cost summary functionality changes according to your Datadog usage as a sing
 
 {{< img src="account_management/plan_and_usage/multiorg-current-month-historical-costs.png" alt="Screenshot of the current month's Cost Summary for a parent organization, showing the overall month-to-date cost, projected cost, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes." >}}
 
-View historical costs by toggling to previous months.
+View historical costs by toggling back to previous months or use the date dropdown to view costs over 1,3, 6 or 12 months.
```
Comment (aliciascott) [inline content/en/account_management/plan_and_usage/cost_details.md:48]:
```suggestion
View historical costs by toggling back to previous months, or use the date dropdown to view costs over 1,3, 6 or 12 months.
```

File: content/en/account_management/plan_and_usage/cost_details.md [hunk 2]
```diff
@@ -75,13 +75,17 @@ As a sub-organization, you can view the costs for your organization only. This r
 
 {{< img src="account_management/plan_and_usage/suborg-cost-trends.png" alt="Screenshot of the current month's Cost Summary for a sub-organization, showing the overall month-to-date cost, projected cost, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes.">}}
 
+View historical costs by toggling back to previous months or use the date dropdown to view costs over 1,3, 6 or 12 months.
```
Comment (aliciascott) [inline content/en/account_management/plan_and_usage/cost_details.md:78]:
```suggestion
View historical costs by toggling back to previous months, or use the date dropdown to view costs over 1,3, 6 or 12 months.
```

File: content/en/account_management/plan_and_usage/cost_details.md [hunk 3]
```diff
@@ -46,17 +45,18 @@ The cost summary functionality changes according to your Datadog usage as a sing
 
 {{< img src="account_management/plan_and_usage/multiorg-current-month-historical-costs.png" alt="Screenshot of the current month's Cost Summary for a parent organization, showing the overall month-to-date cost, projected cost, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes." >}}
 
-View historical costs by toggling to previous months.
+View historical costs by toggling back to previous months or use the date dropdown to view costs over 1,3, 6 or 12 months.
 
-{{< img src="account_management/plan_and_usage/multiorg-prior-month-historical-costs.png" alt="Screenshot of a previous month's Cost Summary for a parent organization, showing the overall cost for the month, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes." >}}
+{{< img src="static/images/account_management/plan_and_usage/parent-org-multi-month-cost-changes.png" alt="Screenshot of a parent organization's historical costs over a three month period, showing the overall cost for the month, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes." >}}
```
Comment (aliciascott) [inline content/en/account_management/plan_and_usage/cost_details.md:50]:
```suggestion
{{< img src="account_management/plan_and_usage/parent-org-multi-month-cost-changes.png" alt="Screenshot of a parent organization's historical costs over a three month period, showing the overall cost for the month, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes." >}}
```

This image isn't rendering on your staging preview, we actually don't need to preface it with `/static/images`

File: content/en/account_management/plan_and_usage/cost_details.md [hunk 4]
```diff
@@ -75,13 +75,17 @@ As a sub-organization, you can view the costs for your organization only. This r
 
 {{< img src="account_management/plan_and_usage/suborg-cost-trends.png" alt="Screenshot of the current month's Cost Summary for a sub-organization, showing the overall month-to-date cost, projected cost, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes.">}}
 
+View historical costs by toggling back to previous months or use the date dropdown to view costs over 1,3, 6 or 12 months.
+
+{{< img src="static/images/account_management/plan_and_usage/suborg-multi-month-cost-changes.png" alt="Screenshot of a sub organization's historical costs over a six month period, showing the overall cost for the month, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes." >}}
```
Comment (aliciascott) [inline content/en/account_management/plan_and_usage/cost_details.md:80]:
```suggestion
{{< img src="account_management/plan_and_usage/suborg-multi-month-cost-changes.png" alt="Screenshot of a sub organization's historical costs over a six month period, showing the overall cost for the month, a graph with cumulative cost breakdowns, and a summary table including month-over-month cost changes." >}}
```

Comment (aliciascott) [reply content/en/account_management/plan_and_usage/cost_details.md:80]: same with this one

**Review by aliciascott — CHANGES_REQUESTED** [review]
Hi @plaxidesnjeridd just a few small suggestions and some fixes for you as the images aren't rendering, let me know once fixed and i'll get this merged for you :)

**Comment by plaxidesnjeridd** [comment]
@aliciascott Thanks for looking through. I just committed changes.

**Review by aliciascott — APPROVED** [review]
looks good thanks!

**Pull request merged**